packages/teams-js/src/private/hostEntity.ts

```diff
@@ -0,0 +1,324 @@
+import { sendMessageToParentAsync } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { ApiName, ApiVersionNumber, getApiVersionTag } from '../internal/telemetry';
+import { ErrorCode, SdkError } from '../public';
+import {
+  ConfigurableTabInstance,
```

Comment:
Thank you for making these 3 new interfaces (ConfigurableTabInstance, HostEntityTabInstance, HostEntityTabInstances). I'd like them to be moved into the `hostEntity.tab` namespace for 2 reasons:

1) Putting them in the public interfaces file makes our documentation messy, and ultimately I'd like to remove as many things as possible. Anything put in that file gets put in a giant doc list of every exported public interface, no matter who uses it. If you move them into this namespace the interface documentation will instead be located next to the documentation for this new capability, which is where developers would expect to see it.

2) This capability is private and undocumented, but the interfaces are currently in the public, documented interfaces file. This means that developers will get to see the interface documentation but not what it is used for.